### DISCUSSION TITLE
Distributing workload equally across a resources pool for a single operation (Community Edition)

### DISCUSSION THREAD

**Discussion by ctpntn** [frePPLe/frepple#675] [Q&A]
Hello everyone,

I’m currently using the frePPLe Community Edition and having trouble configuring an operation to distribute workload equally across all resources in a group. Here’s the context:
I need to set up an operation (for example, “Lamination”) that requires a team of operators working simultaneously. Ideally, I want frePPLe to treat this group of resources as a single unit and split the operation’s workload across all operators, reducing the operation’s total duration. However, I’ve noticed that frePPLe defaults to selecting only one resource from the group, regardless of the group’s total capacity.

I’ve tried the following without success:

1. Setting the individual resources to “default” type and grouping them under a “bucketized” group, assuming it would treat the team as a collective resource pool.
2. Specifying the quantity of resources in the Resource Requirements for the operation.
3. Modifying production rates to reflect total capacity with multiple operators.

Despite these attempts, the operation’s start and end times do not adjust based on the increased capacity from additional operators in the group.
Does anyone have experience configuring operations to utilize multiple resources concurrently in the Community Edition? Or is this functionality only available in the Enterprise version?
Any guidance, tips, or documentation references would be greatly appreciated!

Thank you!

**Comment by jdetaeye** [comment]
Frepple's planning algorithm will not dynamically change the number of assigned operators in function of their workload.   

You can however:
- In the Enterprise Edition, you can manually add or remove operators. The duration of the manufacturing order will be changing then.
- You can also use size-based alternates, where the chosen alternate varies with the requirement size.  Eg if the requirement is for less 100 pieces, I use an operation with 1 operator. If the requirements is between 100 and 300 pieces, I use another operation with 2 operators, etc...   Use the size-minimum and size-maximum fields on the operation, and set the number of operators needed in the operationresource table.
